docs/index.html

```diff
@@ -16,17 +16,33 @@
     window.$docsify = {
         name: 'DAOstack Arc Docs',
         repo: 'https://github.com/daostack/daostack',
+        homepage: 'https://raw.githubusercontent.com/daostack/daostack/master/README.md',
```

Comment:
Why homepage is raw ?

Reply:
As seen [here](https://docsify.js.org/#/configuration?id=homepage)